src/fga/fga-live-test.spec.ts

```diff
@@ -0,0 +1,1136 @@
+import { disableFetchMocks, enableFetchMocks } from 'jest-fetch-mock';
+
+import { WorkOS } from '../workos';
+import { CheckOp, WarrantOp } from './interfaces';
+
+describe.skip('FGA Live Test', () => {
```

Comment:
Is this `.skip` intentional? I'm guessing yes, given the `live` naming, but just wanted to double-check.

Reply:
Correct. Ideally we'll run this in CI in the future but right now we're only running it manually